docs-new/docs/guides/assets/metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: available-cpus
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-prometheus
+  query: "sum(kube_node_status_capacity{resources`cpu`})"
+  fetchIntervalSeconds: 10
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: availability-slo
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-dynatrace
+  query: "func:slo.availability_simplenodeservice"
+  fetchIntervalSeconds: 10
```

Comment:
```suggestion
  fetchIntervalSeconds: 10

```

Reply:
@odubajDT are you suggesting I remove the `fetchIntervalSeconds`? If yes, that would mean the docs will also be updated by removing this line

> Each of these are configured to fetch data every 10 seconds but you could configure a different fetchIntervalSeconds value for each metric.

Reply:
I was suggesting to add a newline, the `fetchIntervalSeconds` is completely ok here. Or am I missing something ?

Reply:
Oh! I got a little confused cus' I already fixed that before your review came in. You can review again to see the changes.